visallo/visallo
# add open source gpw required intents to minimal owl

Testing Instructions:
- Following the [getting started](http://docs.visallo.org/getting-started.html) instructions, make sure that all of the open source gpws requiring intents work (email extractor, phone number extractor, and zip code extractor)

Points of Regression: N/A

CHANGELOG
Added: Missing required intents into minimal owl file for GPWs
